config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,3 @@
+require 'syslog/logger'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.